src/services/componentService.js

```diff
@@ -14,24 +14,22 @@ const queryBuilder = (name, target = '', query) => ({
 const filesQuery = `
   ... on Tree {
     entries {
-      ... on TreeEntry {
-        name
-        object {
-          ... on Blob {
-            text
-          }
+      name
+      object {
+        ... on Blob {
+          text
         }
       }
     }
   }`
 
 export const getComponentFiles = async (componentType, component) => {
-  const response = await api.post('', queryBuilder('componentsFiles', `:${componentType}/${component}`, filesQuery))
+  const response = await api.post('', queryBuilder('componentsFiles', `/${componentType}/${component}`, filesQuery))
```

Comment:
You can use [this](https://www.npmjs.com/package/qs) but it's ok

Reply:
That won't work, I need to pass the query as a parameter, not an object